tests/e2e/tools/lib/columns.js

```diff
@@ -43,6 +54,11 @@ async function checkColumnsBlock(page, style) {
       const link = column.locator('a.standalone-link');
       expect(link).toHaveText(`Link ${index + 1}`);
       expect(link.getAttribute('href')).toBe(TEST_LINKS[index]);
+    } else if (style === 'Icons') {
```

Comment:
It looks like these are exactly the same checks as for the `Images` styles, maybe we should handle them in the same condition in order to avoid duplication?

Reply:
Yes, I updated the condition. 🙂